storage/remote/queue_manager.go

```diff
@@ -1921,11 +1924,35 @@ func populateV2TimeSeries(symbolTable *writev2.SymbolsTable, batch []timeSeries,
 			pendingData[nPending].Metadata.UnitRef = symbolTable.Symbolize(d.metadata.Unit)
 			nPendingMetadata++
 		} else {
+			metricType := writev2.Metadata_METRIC_TYPE_UNSPECIFIED
+			var unit string
+			if enableTypeAndUnitLabels {
+				typeValue := d.seriesLabels.Get("__type__")
+				unit = d.seriesLabels.Get("__unit__")
```

Comment:
I could be wrong here, but I'm assuming that I can access this labels, because @ArthurSens recently added this line ⬇️, where when the feature flag is enabled, the labels __type__ and __unit__ will be populated.

https://github.com/prometheus/prometheus/blob/294f36e80261bb2e2364aa3d0700ebaf20a17bca/storage/remote/otlptranslator/prometheusremotewrite/helper.go#L206-L214